vignettes/lintr.Rmd

```diff
@@ -194,29 +195,49 @@ defaults_table <- data.frame(
 knitr::kable(defaults_table)
 ```
 
+Another way to customize linters is by specifying tags in `linters_with_tags()`.
+The available tags are listed below:
+
+```{r}
+available_tags(packages = "lintr")
+```
+
+You can select tags of interest to see which linters are included:
+
+```{r}
+linters <- linters_with_tags(tags = c("package_development", "readability"))
+names(linters)
+```
+
+You can include tag-based linters in the configuration file, and customize them further:
+
+```r
```

Comment:
I just noticed, these config examples should be ```` ``` yaml ```` for consistency with ll. 77 and clearer separation of R code examples and config examples.

Reply:
Good idea. Done.